### PR TITLE
Profile menu routing

### PR DESCRIPTION
**Pull request by cvogitgud**
Added pages for Profile/Account, Settings, Favorites, Reviews, Billing, as well as a general Not Found page.

### CODE REVIEW

File: app/components/ProfileMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                    {(await isAuthenticated()) ? (
+                        <LogoutLink>
+                            <div className="flex items-center">
+                                <LogOut className="size-4 mr-2" /> Log out
+                            </div>
+                        </LogoutLink>
+                    ) : (
+                        <LoginLink>
+                            <div className="flex items-center">
+                                <LogIn className="size-4 mr-2" /> Log in
```
[the remark this answers is not in the document]
Comment (cindyquach12) [reply app/components/ProfileMenu.tsx:88]: or does it even need the Log in text at all?
```suggestion
                                <LogIn className="size-4 mr-2" />
```

File: app/not-found.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default function NotFound() {
```
Comment (cindyquach12) [inline app/not-found.tsx:1]: Let's change this to use `FunctionComponent` and same for the rest of the files

**Review by cindyquach12 — CHANGES_REQUESTED** [review]
Great job!! Just a few comments :)

**Comment by cindyquach12** [comment]
also, you need to resolve merge conflicts on this branch, lmk if you need help w/ that

File: app/components/ProfileMenu.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (cvogitgud) [reply app/components/ProfileMenu.tsx:88]: LogIn is an icon from lucide-react!

File: app/components/NavBar.tsx [hunk 3]
```diff
@@ -1,45 +1,43 @@
...
-import { getKindeServerSession } from '@kinde-oss/kinde-auth-nextjs/server';
-import { ThemeToggle } from './ThemeToggle';
+    RegisterLink,
+    LoginLink,
+} from "@kinde-oss/kinde-auth-nextjs/components";
+import { getKindeServerSession } from "@kinde-oss/kinde-auth-nextjs/server";
+import { ThemeToggle } from "./ThemeToggle";
+import { ProfileMenu } from "./ProfileMenu";
 
 export async function NavBar() {
```
[the remark this answers is not in the document]
Comment (cvogitgud) [reply app/components/NavBar.tsx:11]: Got it.

File: app/components/ProfileMenu.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (cvogitgud) [reply app/components/ProfileMenu.tsx:88]: I guess I could make it clearer and use a p or span tag?

**Comment by cvogitgud** [comment]
merge conflict in ProfileMenu resolved

**Review by cindyquach12 — APPROVED** [review]

**Pull request merged**